platforms/ide/tooling-api/src/main/java/org/gradle/tooling/internal/protocol/problem/InternalProblemDetailsVersion2.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.tooling.internal.protocol.problem;
+
+public interface InternalProblemDetailsVersion2 {
+
```

Comment:
@donat I removed the second one in `org.gradle.tooling.internal.protocol`, is that ok?

Reply:
That make sense.